source/key-concepts/input_data.md

```diff
@@ -1,198 +1,116 @@
+- `salary` is a Person entity variable defined on a monthly basis,
+- `rent` is a Household entity variable defined on a monthly basis as well.
 
-```python
-In [6]: data.person_salary
-Out[6]:
-0     2694
-1     2720
-2     1865
-3     1941
-4     2393
-5     3008
-6     2286
-7     3386
-8     2929
-9     3981
-10    3643
-11    2078
-Name: person_salary, dtype: int64
-```
-
-
-3. Build a simulation according to your data's length:
-
-```python
-In [7]: from openfisca_core.simulation_builder import SimulationBuilder
 
-In [8]: simulation = SimulationBuilder().build_default_simulation(tax_benefit_system, length)
-```
+### Data
 
-4. Configure the simulation and calculate the [income_tax](https://demo.openfisca.org/legislation/income_tax) variable for all persons on the same period:
+Data sets describe multiple people situations. It could define a whole population.
+This data could come from a survey with aggregated data, data files extracted from a database, etc.
 
-```python
-In [9]: import numpy as numpy
+> Here is a [survey example](https://www.casd.eu/en/source/tax-and-social-incomes-survey/). It typically goes from 50 000 households to 500 000.
 
-In [10]: period = '2018-01'
+Here is a minimal example of data (in CSV format):
 
-# match data from the 'person_salary' column
-# with the 'salary' variable of our yet to be country's tax-benefit system
-In [11]: simulation.set_input('salary', period, numpy.array(data.person_salary))
-
-In [12]: income_tax = simulation.calculate('income_tax', period)
+```csv
+person_id,household_id,person_salary,age
+1,a,2694,40
+2,a,2720,43
+3,b,3865,45
+4,b,1300,23
+5,c,0,12
+6,c,0,14
+7,c,2884,44
+12,e,1200,38
+8,d,3386,27
+9,d,2929,28
+10,e,0,10
 ```
 
-5. You are all set! The `income_tax` has been calculated for each person on your `data.csv` file.
+As for the *test case* content, you will need the following information:
+- unique indentifiers for persons and *group entities*
+  > like `person_id` and `household_id` columns information in the CSV example
+- if you have multiple [entities](./key-concepts/person,_entities,_role.md) types (persons, households, ...), you need to know how your persons list is dispatched over your *group entities*
+  > in CSV example, every `person_id` is associated with a `household_id` on the same line
+- the name of the corresponding variable in your model for every set of values 
+  > `person_salary` values become [salary](https://demo.openfisca.org/legislation/salary) values in `OpenFisca-Country-Template` model
+- the period and entity for every set of values
+  > `person_salary` and `age` belong to *Person* entity  
+  > the definition period isn't in the CSV file but it might, for example, come from the CSV creation time and be identical for the whole data set.
 
-Persons' order is kept:
 
-```python
-In [13]: data.person_id.values
-Out[13]: array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12])
-```
+### Simulation
```

Comment:
There is already a [full page](https://openfisca.org/doc/key-concepts/simulation.html) about simulations. This paragraph seem a little out of place.

Reply:
Deleted.